model.rb

```diff
@@ -12,7 +12,15 @@
 Sequel::Model.plugin :singular_table_names
 Sequel::Model.plugin :subclasses unless ENV["RACK_ENV"] == "development"
 Sequel::Model.plugin :column_encryption do |enc|
-  enc.key 0, Config.clover_column_encryption_key
+  key = Config.clover_column_encryption_key
+  unless Config.kms_encrypt_clover_column_encryption_key_region.nil?
```

Comment:
`nil`/`false` difference is not important in this case, so it's better to be idiomatic:

```suggestion
  if Config.kms_encrypt_clover_column_encryption_key_region
```